lib/vtls/schannel.c

```diff
@@ -884,14 +884,13 @@ schannel_recv(struct connectdata *conn, int sockindex,
                            size, &nread);
     /* check for received data */
     if(*err != CURLE_OK)
-      ret = -1;
+      return -1;
     else {
```

Comment:
The surrounding code makes use of the else in order to show the code path and structure using indentation. Yes, the else-branch is not needed, but just in case the return is removed from the if-branch, the logic would stay intact.